src/main/java/data/schedule/DayList.java

```diff
@@ -183,13 +187,20 @@ public void clearDayPlan(String userArgument) throws InvalidScheduleException {
      */
     public void clearAllSchedule() {
         for (int i = 0; i < NUMBER_OF_SCHEDULE_DAYS; i += 1) {
-            scheduleList[i] = null;
-            dayList[i] = null;
+            clearPlan(i);
             assert (scheduleList[i] == null);
             assert (dayList[i] == null);
         }
     }
 
+    /**
+     * Remove plan on the stated day.
+     */
+    public void clearPlan(int dayNumber) {
+        scheduleList[dayNumber] = null;
+        dayList[dayNumber] = null;
+    }
```

Comment:
Perhaps a better method name is clearDay()? because clear day means clear the plan?